Makefile

```diff
@@ -0,0 +1,4 @@
+test:
+	@./tests/test_testimony.sh | sed -r "s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g" | diff tests/sample_output.txt -
```

Comment:
Remove color codes if the output is colored otherwise it will fail. This is actually a way to normalize the output and make test pass with and without colored output.